pg_metadata/pg_metadata.py

```diff
@@ -83,8 +86,12 @@ def check_invalid_connection_names(self):
         msg = QMessageBox()
         msg.setIcon(QMessageBox.Warning)
         msg.setWindowTitle(tr('PgMetadata: Database connection(s) not available'))
-        msg.setText(tr(f'{n_invalid} connection(s) listed in PgMetadata’s settings are invalid or no longer available: {invalid_text}'))
-        msg.setInformativeText(tr('Do you want to remove these connection(s) from the PgMetadata settings? (You can also do this later with the ”Set Connections” tool.)'))
+        msg.setText(tr(f'{n_invalid} connection(s) listed in PgMetadata’s settings are invalid or '
+                       f'no longer available: {invalid_text}'))
+        # FIXME: inserted linebreak because Flake8 failed, but will this mess with Transifex?
```

Comment:
Not sure what you mean, but for transifex, this looks all good !

You used fstring, I was careful before because of the QGIS minimum version for Python, CF my previous message. So I will increase just to be sure.

We have the habits to always keep indentations of 4 spaces.

For instance instead of your code : 

```python
msg.setText(tr(f'{n_invalid} connection(s) listed in PgMetadata’s settings are invalid or '
               f'no longer available: {invalid_text}'))  # this line is not a multiple of 4 indents
```
we would do
```python
 msg.setText(tr(
    f'{n_invalid} connection(s) listed in PgMetadata’s settings are invalid or '
    f'no longer available: {invalid_text}'))
```

Reply:
Thanks for the hint!

With Transifex, I’ve wondered if two string constants next to each other (which are concatenated in Python) would appear as a single string in the translations list on Transifex. I assumed they would, because they’re within a single tr() call, but wasn’t sure.

Reply:
I’ll add translation as soon as the strings are uploaded to Transifex. Will this happen only after a new version has been released “officially”?

Also, maybe we could address #120 while working on the connections code. ;-)

Reply:
> I’ll add translation as soon as the strings are uploaded to Transifex. Will this happen only after a new version has been released “officially”?

Translations are pushed on transifex automatically after the merge of this PR. So @mdouchin and I will let you some times to translate in Transifex (and for us too in French) before we make the new release.

Reply:
Should I update my PR for the indentation?

Reply:
Can you fix the  indent indeed ?